src/js/components/Button/StyledButton.js

```diff
@@ -122,6 +132,20 @@ const plainStyle = props => css`
   text-align: inherit;
 `;
 
+const normalizedStyle = props => css`
+  ${backgroundStyle(
+    normalizeColor(
+      props.colorValue || props.theme.button.normalized.color || undefined,
+      props.theme,
+    ),
+    props.theme,
+    props.theme.button.color,
+  )}
+  border: none;
```

Comment:
would this affect the styling of normalized + focus?

Reply:
The focus should still appear around the button as normal.